common/customizable_localization/ATHA_name_gen.txt

```diff
@@ -0,0 +1,107 @@
+﻿#ATHAUIName = {
+#    type = character
+#    text = {
+#        trigger = {
+#            has_culture = culture:celtic      Or culture group if sharing one name format
+#        }
+#        localization_key = UIName_celtic
+#    }
+#    text = {
+#        fallback = yes
+#        localization_key = UIName_fallback
+#    }
+#}
+#  ^^^^ need one for each used 'GetName'
+
+
+NameCheck = {
+    type = character
+    text = {
+        trigger = {
+            has_culture = culture:celtic
+            OR = {
+                AND = {
+                    exists = dynasty #shorten lists for highborns
+                    dynasty = {
+                        any_in_list = {
+                            variable = celtic_name_support
+                            this = root
+                        }
+                    }
+                }
+                any_in_global_list = { #longer lists for lowborns
+                    variable = celtic_name_support
+                    this = root
+                }
+            }
+        }
+        localization_key = ATHA_supported_name
+    }
+    text = {
+        fallback = yes
+        localization_key = ATHA_unsupported_name
+    }
+}
+PatCheck = {
+    type = character
+    text = {
+        trigger = {
+            exists = father
+            has_culture = culture:celtic #
+            father = {
+                save_temporary_scope_as = daddy #we want to check that the dad has support
+                OR = {
+                    AND = {
+                        exists = dynasty #shorten lists for highborns
+                        dynasty = {
+                            any_in_list = {
+                                variable = celtic_name_support
```

Comment:
I assume these characters are added to the list when they are born or die?

Reply:
Yeap, I usually give it to them on birth. You just have to set up a sgui to take it away for the event window naming widget